torch/_export/converter.py

```diff
@@ -465,9 +503,8 @@ def convert_prim_GetAttr(self, node: torch._C.Node):
         output_name = node.output().debugName()
         self.name_to_attribute_fqn[output_name] = attr_fqn
 
-        attr_value = node.output()
         if self.is_top_level_graph():
-            if attr_value.type().annotation_str == "Tensor":
```

Comment:
I find this check to be not reliable, e.g., sometimes this is `Optional[Tensor]`.